hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/connector/WriteMapP.java

```diff
@@ -76,7 +75,7 @@ public boolean tryProcess() {
     @Override
     public void process(int ordinal, @Nonnull Inbox inbox) {
         checkFailure();
-        if (Util.tryIncrement(numParallelOps, 1, parallelOpsLimit)) {
+        if (Util.tryIncrement(numParallelOps, 1, MAX_LOCAL_PARALLEL_OPS)) {
```

Comment:
We can also replace this with `AtomicBoolean opPending` and `if (opPending.compareAndSet(false, true))`

Reply:
I'll keep it for now in case we decide to increase the limit in the future, or make it configurable in some other way.